package.json

```diff
@@ -10,16 +10,27 @@
     "localhost": "electron electron --debug=5858 --devtools --enable-logging --env=http://localhost:8888",
     "test": "eslint electron && electron-mocha tests",
     "rebuild-native-modules": "electron-rebuild -m ./electron/node_modules -f",
-    "deb64": "electron-installer-debian --src wrap/build/Wire-linux-x64/ --arch amd64 --config linux.json",
-    "deb32": "electron-installer-debian --src wrap/build/Wire-linux-ia32/ --arch x32 --config linux.json"
+    "linux32": "build --ia32",
```

Comment:
I suggest to add `-l`, otherwise if you run it on macOS, it will be macOS, not Linux :)